utils/string_replacer/out_strings.csv

```diff
@@ -0,0 +1,97 @@
+Num	String

```

Comment:
I assume this file is an output of your script. This and any other output files must not be commited to the repo.